test/proposal/space_maker_test.rb

```diff
@@ -44,29 +44,30 @@ def probed_partition(name)
       settings.root_device = "/dev/sda"
       settings
     end
-    let(:volumes) { vols_list(vol1) }
+    let(:proposed_partitions) { [partition1] }
+    let(:keep) { [] }
     let(:analyzer) { Y2Storage::DiskAnalyzer.new(fake_devicegraph) }
-    let(:lvm_helper) { Y2Storage::Proposal::LvmHelper.new(Y2Storage::PlannedVolumesList.new) }
+    let(:lvm_helper) { Y2Storage::Proposal::LvmHelper.new([]) }
     let(:windows_partitions) { Hash.new }
 
     subject(:maker) { described_class.new(fake_devicegraph, analyzer, lvm_helper, settings) }
 
     context "if the only disk is not big enough" do
       let(:scenario) { "empty_hard_disk_mbr_50GiB" }
-      let(:vol1) { planned_vol(mount_point: "/1", type: :ext4, desired: 60.GiB) }
+      let(:partition1) { proposed_partition(mount_point: "/1", type: :ext4, disk_size: 60.GiB) }
 
       it "raises a NoDiskSpaceError exception" do
-        expect { maker.provide_space(volumes) }
+        expect { maker.provide_space(proposed_partitions, partitions_to_keep: keep) }
```

Comment:
Has `partitions_to_keep` become mandatory or something like that? In this new version of the test it looks more relevant than before (it used to be omitted when not relevant).